semgrep-core/src/parsing/tree_sitter/Parse_csharp_tree_sitter.ml

```diff
@@ -1003,7 +1003,7 @@ and switch_expression_arm (env : env)
   in
   let _v3 = token env v3 (* "=>" *) in
   let v4 = expression env v4 in
-  (v2, v4)
+  G.CasesAndBody ([ G.Case (fake "case", v2) ], G.exprstmt v4)
```

Comment:
sometimes when those expressions appear in multiple xxx_to_generic.ml, I add them to AST_generic_helpers.ml or even AST_generic.ml (e.g., AST_generic.interpolated), so that way if we decide at some point to change again, we just need to modify one function.